Hi3Helper.Core/Classes/Data/InvokeProp.cs

```diff
@@ -203,12 +203,18 @@ public struct WindowRect
         [DllImport("user32.dll")]
         public extern static uint SetWindowLong(IntPtr hwnd, int index, uint value);
 
+        [DllImport("user32.dll")]
+        public extern static IntPtr SetWindowLongPtr(IntPtr hwnd, int index, IntPtr value);
+
         [DllImport("user32.dll")]
         public static extern IntPtr FindWindowEx(IntPtr hwndParent, IntPtr hwndChildAfter, string lpszClass, string lpszWindow);
 
         [DllImport("user32.dll")]
         public static extern bool DestroyWindow(IntPtr hwnd);
 
+        [DllImport("user32.dll")]
```

Comment:
We probably want to organize all externs used in InvokeProp but we can do that at next refactor.